compiler/noirc_frontend/src/parser/parser/types.rs

```diff
@@ -235,6 +245,23 @@ pub(super) fn slice_type(
 }
 
 pub(super) fn type_expression() -> impl NoirParser<UnresolvedTypeExpression> {
+    type_expression_inner().try_map(UnresolvedTypeExpression::from_expr)
+}
```

Comment:
Can you remove the `pub(super)` on these functions? They were here before but I just looked and we don't actually use these outside this file.

Reply:
done